core/sql/comexe/ComTdbUdr.h

```diff
@@ -322,8 +325,12 @@ class ComTdbUdr : public ComTdb
 
   Int32 javaDebugPort_;                                   // 236-239
   Int32 javaDebugTimeout_;                                // 240-243
+  Int64 libraryRedefTime_;                                // 244-251
+  NABasicPtr libraryBlobHandle_;                          // 252-259
+  NABasicPtr librarySchName_;                             // 260-267
+  Int32 libraryVersion_;                                  // 268-271
   // Make sure class size is a multiple of 8
-  char fillerComTdbUdr2_[28];                             // 244-271
+  char fillerComTdbUdr2_[24];                             // 272-295
```

Comment:
Just a question to understand the code a bit better.  Are there rules on how big a filler of a structure is?  The new field size adds up to 28 which would take up the entire filler.  So a 24 extra bytes were added to allow for extra expansion.  Why was 24 chosen?

Reply:
Just has to be a multiple of 8. These were more relevant when we had  versioning support. It's not the case now but we try to keep some filler anyway.